### PR TITLE
updating website for version 1.1.0

### PR DESCRIPTION
**Pull request by b00f**
This PR updates website fro version 1.1.0

This PR mainly updates the URLs in the user guides and removes EXIF information from pictures.

### CODE REVIEW

File: website/_posts/2024-04-14-release-1-1-0.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  without the need to sync the entire blockchain.
+
+This release is named after the city of [Hanoi](https://en.wikipedia.org/wiki/Hanoi), \
+and it was unanimously selected by the Pactus community.
+
+![Pactus Version 1.1.0 (Hanoi)]({% image_url %}/pactus-1-1-0-hanoi.png)
+
+## Highlights
+
+TODO
```
Comment (Mr-HoDL58) [inline website/_posts/2024-04-14-release-1-1-0.md:35]: You are still working on this?

**Review by Mr-HoDL58 — COMMENTED** [review]
Why are we deleting images and then adding them again?

File: website/_posts/2024-04-14-release-1-1-0.md (hunk 1 above)
Comment (b00f) [reply website/_posts/2024-04-14-release-1-1-0.md:35]: Yes...

Comment (b00f) [reply website/_posts/2024-04-14-release-1-1-0.md]: Done...

**Comment by b00f** [comment]
> Why are we deleting images and then adding them again?
No, We remove EXIF data from images.

**Pull request merged**